sensu-plugins-sensu.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsSensu::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '~> 2.4'
```

Comment:
This is a `### Breaking Change` and needs to be called out as such in the changelog. Something like this:
```
### Breaking Change
- bumped dependency of `sensu-plugin` to 2.x you can read about it  [here](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v145---2017-03-07)
```

Reply:
Good point, I didn't realize a change in dependency would cause a breaking change, updated changelog as you suggested.